service/component-samples/manufacturer/src/main/java/org/fido/iot/sample/ManufacturerContextListener.java

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fido.iot.sample;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.security.cert.Certificate;
+import java.util.List;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletContextEvent;
+import javax.servlet.ServletContextListener;
+import javax.sql.DataSource;
+import org.apache.commons.dbcp2.BasicDataSource;
+import org.fido.iot.certutils.PemLoader;
+import org.fido.iot.protocol.CloseableKey;
+import org.fido.iot.protocol.Composite;
+import org.fido.iot.protocol.Const;
+import org.fido.iot.protocol.CryptoService;
+import org.fido.iot.protocol.DiServerService;
+import org.fido.iot.protocol.DiServerStorage;
+import org.fido.iot.protocol.MessageDispatcher;
+import org.fido.iot.protocol.MessagingService;
+import org.fido.iot.storage.CertificateResolver;
+import org.fido.iot.storage.DiDbManager;
+import org.fido.iot.storage.DiDbStorage;
+
+/**
+ * Device Initialization servlet Context Listener.
+ */
+public class ManufacturerContextListener implements ServletContextListener {
+
+  private static final String mfgKeyPem = "-----BEGIN CERTIFICATE-----\n"
```

Comment:
In a subsequent PR, make this configurable.

Reply:
Agreed.